### ISSUE TITLE
Making Model nodes transparent.

### ISSUE THREAD

**Issue by devond5** [CesiumGS/cesium#8919]
Hello Cesium team,
I have an employer who we are visualizing things that go up into space, like deep space. These things have many moving parts/nodes from a 3d model both externally and internally that we transform. The outer nodes are easily seen when they move, but the internal ones are not. Here's the problem. We can remove those nodes with show and hide which is great but say you want to keep an eye on the outside movements while seeing the inner workings move. I have posted this question to the forums and so I see many others have as well.

This doesn't just apply for space exploration cases, what if you want to see into only a section of a building, or check a satellites internal parts that can be telemetry driven, or if your 3d tile of a city has multiple node layers you could make the top skyscrapers transparent to see the more internal architecture, or if you want to see into santa's sack?

I know you already have the capability to grab individual nodes and show hide them from a model, you would only have to change the alpha or transparency of that node.

We have a way now to do this but it requires setting the model opaque, diving into the gltf, finding the nodes we don't want to be transparent by traversing the heiarchy, determining the child nodes we want to affect, and deleting the translucent command for all other nodes.
let a = primitive._runtime.nodesByName[node.name];
let b = a.commands[i];
delete b.translucentCommand;

The big problem with this is that you can only make one node transparent at a time and always to the same level of transparency based off of the overall models transparency. For instance what if some thing has multiple layers to it you would have to bring the alpha to 0.36 to see through it wheras that would be too much for another node that would only need to be brought to 0.85.

**Comment by lilleyse** [comment]
@devond5 agreed this would be useful functionality to have. There's some more information in https://github.com/CesiumGS/cesium/issues/2387.

The only workaround I can think of at the moment is to create a 3D Tiles variant of your model composed of a single [Batched 3D Model](https://github.com/CesiumGS/3d-tiles/blob/master/specification/TileFormats/Batched3DModel/README.md) using feature ids to identify components of the model that you want to individually color or apply alpha to. This capability is part of the 3D Tiles implementation but not the Model implementation yet.

**Comment by devond5** [comment]
Is there any progress with this?

**Comment by OmarShehata** [comment]
@devond5 we aren't actively working on this at the moment, but if you'd like to submit a pull request we'd be happy to consider it! See the contributing guide here: https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#getting-started-contributing
